models/UserCartDetail.js

```diff
@@ -0,0 +1,25 @@
+/* jshint indent: 1 */
+
+module.exports = function(sequelize, DataTypes) {
+	return sequelize.define('UserCartDetail', {
+		cart_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: false,
+			primaryKey: true,
+			references: {
+				model: 'UserCartHeader',
+				key: 'id'
+			}
+		},
+		product_id: {
+			type: DataTypes.INTEGER(11),
+			allowNull: true
+		},
+		quantity: {
```

Comment:
Set a default to zero especially for Integer type fields.

Reply:
instead, made it NOT NULL. Since its order quantity, default of 0 doesnt make sense